keyboards/kuro/kuro65/readme.md

```diff
@@ -0,0 +1,41 @@
+# Kuro 65
+<img align="right" width="118" height="207" src="https://user-images.githubusercontent.com/28592326/219898869-d1d8c107-8534-48e9-9dfd-22592b35853f.png">
+A 65% keyboard with an ISO layout, which fits the TOFU65 case (and similar) by KBDFans.
+
+Keyboard maintainer: [0x544D](https://github.com/0x544D)<br>
+Hardware availability: [0x544D/kuro65](https://github.com/0x544D/kuro65)
+
+## Features
+
+- Hot-swappable ISO layout
+- Per key RGB LED (south facing)
+- VIA compatible
+- USB-C connector
+- Reset button on the back of the PCB
+- Compatible with TOFU65 and similar cases 
+
+## Pictures
+
+![kuro65_pcb](https://user-images.githubusercontent.com/28592326/225080112-e31c2d8d-801e-48e6-b4a8-996fe5636419.jpeg)
+
+![kuro65_case](https://user-images.githubusercontent.com/28592326/225080143-74d2be9f-fc1b-4c57-8ad6-8ea516279ff8.jpeg)
```

Comment:
These images should be rehosted on Imgur. And generally HTML tags should be avoided in markdown.

Reply:
I hosted all images on imgur and adjusted the README.md accordingly.

Thank you very much for the review!